docs/learn-identity.md

```diff
@@ -22,11 +22,11 @@ information, so the data must be less than that. When inputting the data manuall
 The easiest way to add the built-in fields is to click the gear icon next to one's account and
 select "Set on-chain identity".
 
-![Gear icon provides the option to set identity](/img/identity/01.jpg)
+![Gear icon provides the option to set identity](/img/identity/new1.jpg)
```

Comment:
Hm these images don't appear when I run 'yarn kusama:start` on my computer

Reply:
<img width="666" alt="Screen Shot 2020-09-14 at 9 50 13 AM" src="https://user-images.githubusercontent.com/36748441/93058442-b1063c00-f66f-11ea-82c0-dac7f395a383.png">

Just these two images don't appear

Reply:
I would also suggest placing the images inside the assets folder, since most of the images are already in there

Reply:
@ansonla3  Did you also want to include new images for https://wiki.polkadot.network/docs/en/maintain-guides-democracy?

Reply:
@krichard410 Please check ;)